sdk/core/Azure.Core.Extensions/samples/Azure.Core.Extensions.Samples.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <RequiredTargetFrameworks>netcoreapp2.1</RequiredTargetFrameworks>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <WarnOnPackingNonPackableProject>false</WarnOnPackingNonPackableProject>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.App" IsImplicitlyDefined="true" />
```

Comment:
Orthogonal to these changes:  I'm not familiar with `IsImplicitlyDefined` and a quick search didn't turn up anything.   I'd be grateful for a quick pointer to a reference or explanation of what it is for.  :)

Reply:
Sooooo, you are not supposed to have package version for `Microsoft.AspNetCore.App` package, SDK would pick the highest patch for you based on the target framework.

Unfortunately, this conflicts with our central version management strategy that produces a warning when something outside of Packages.Data.props set a package version. IsImplicitlyDefined suppresses this warning.